tests/e2e/support/objects/app-files/resource/actions.ts

```diff
@@ -240,7 +252,94 @@ export const createNewFileOrFolder = async (args: createResourceArgs): Promise<v
       await page.goto(page.url())
       break
     }
+    case 'OpenDocument': {
+      // By Default when OpenDocument is created, it is opened with collabora if both app-provider services are running together
+      await createOpenDocumentOrMicrosoftWordFile(args, 'Collabora')
+      break
+    }
+    case 'Microsoft Word': {
+      // By Default when Microsoft Word document is created, it is opened with OnlyOffice if both app-provider services are running together
+      await createOpenDocumentOrMicrosoftWordFile(args, 'OnlyOffice')
+      break
+    }
+  }
+}
+
+const createOpenDocumentOrMicrosoftWordFile = async (
```

Comment:
```suggestion
const createDocumentFile = async (
```

Reply:
`createDocumentFile` am i the only one who feel some discomfort in the word